apps/website/src/pages/docs/dialog.mdx

```diff
@@ -79,6 +79,17 @@ The two buttons at the bottom right are always pinned in that same area, regardl
 
 As specified earlier, full-page dialogs will house various settings options as shown in the example below.
 
+### Placement
+
+A placement dialog can be adjusted to sit in one of the four corners of the page.
+
+<LiveExample src='Dialog.placement.tsx'>
+  <AllExamples.DialogPlacementExample client:load />
+</LiveExample>
+
+Positions for the dialog on the page include:
+`top-left`, `top-right`, `bottom-left`, `bottom-right`
```

Comment:
For consistency sake accross all documentations, I think each value could be on its own line. Refer to `avatar.mdx` for example.

Reply:
done :)